src/components/MDX/TeamMember.tsx

```diff
@@ -40,6 +40,7 @@ export function TeamMember({
 }: TeamMemberProps) {
   if (name == null || title == null || permalink == null || children == null) {
     throw new Error(
+      // @ts-ignore
```

Comment:
What's the error?

Reply:
This was a very funky error (`This binary expression is never nullish`). TypeScript seems to be confused by the chaining of conditionals. I refactored this to be clearer and the error went away.